frontend/src/lib/styles/Difficulty.pcss

```diff
@@ -0,0 +1,9 @@
+.easy{
+    @apply inline-flex items-center gap-x-1.5 py-1.5 px-3 rounded-full text-xs font-medium bg-green-500 text-white;
+}
+.medium{
+    @apply inline-flex items-center gap-x-1.5 py-1.5 px-3 rounded-full text-xs font-medium bg-orange-500 text-white;
+}
+.hard{
+    @apply inline-flex items-center gap-x-1.5 py-1.5 px-3 rounded-full text-xs font-medium bg-red-500 text-white;
+}
```

Comment:
You can refactor these even more since there are a lot of common css here like “inline-flex items-center gap-x-1.5 …” so we don’t repeat the code three times

something like
```

.difficulty {
    @apply inline-flex items-center gap-x-1.5 py-1.5 px-3 rounded-full text-xs font-medium text-white;
}

.easy {
    @apply bg-green-500;
}

.medium {
    @apply bg-orange-500;
}

.hard {
    @apply bg-red-500;
}
```

Then on the html code, 
```
<span class="difficulty easy">Easy</span>

```